### PR TITLE
Animal shelter revisions

### PR DESCRIPTION
**Pull request by samanpwbb**
Adds a cat icon, to represent the `amenity=animal_shelter` osm tag.

100% scale:
Screen Shot 2021-09-24 at 2 08 36 PM

400% scale:
Screen Shot 2021-09-24 at 2 08 45 PM

@mzdraper thanks for getting this started!

@tristen for review

### CODE REVIEW

File: browser.esm.js [hunk 1]
```diff
@@ -1,3 +1,3 @@
...
...55,2.4902,6.5723C3.9685,14.4471,5.8379,15,7.5,15&#xA;&#x9;c1.6656,0,3.535-0.5596,5.0117-1.9395S14.998,9.4868,14.998,6.498c0.0648-1.3953-2.0628-1.3953-1.998,0&#xA;&#x9;c0,2.553-0.7997,4.1149-1.8535,5.0996C10.3731,12.3203,9.4288,12.7084,8.5,12.875V6.8418C9.9607,6.4058,10.9986,5.0642,11,3.5&#xA;&#x9;C11,1.567,9.5,0,7.5,0z M7.5,2C8.3284,2,9,2.6716,9,3.5S8.3284,5,7.5,5S6,4.3284,6,3.5S6.6716,2,7.5,2z\"/>\n</svg>","<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n<svg version=\"1.1\" id=\"hardware\" xmlns=\"http://www.w3.org/2000/svg\" width=\"15\" height=\"15\" viewBox=\"0 0 15 15\">\n  <path d=\"M13.7919,3.2619c0,0-1.676,1.675-2.1163,2.1208c-0.085,0.0861-0.1688,0.1135-0.282,0.0961&#xA;&#x9;c-0.1481-0.0226-0.2974-0.038-0.4462-0.0558c-0.4072-0.0485-0.8145-0.0966-1.2292-0.1458C9.649,4.6852,9.5787,4.1049,9.5177,3.5236&#xA;&#x9;C9.512,3.4689,9.5509,3.3943,9.5925,3.3522c0.5071-0.5134,1.9261-1.9287,2.134-2.136c-0.4508-0.2129-1.2243-0.2968-1.8007-0.2031&#xA;&#x9;c-2.1801,0.3543-3.5112,2.534-2.8206,4.625C7.1432,5.753,7.1194,5.8201,7.0374,5.902C5.1891,7.7454,3.3436,9.5914,1.498,11.4374&#xA;&#x9;c-0.0616,0.0616-0.1231,0.124-0.1779,0.1913c-0.5264,0.6473-0.3873,1.6264,0.2974,2.102c0.6044,0.4197,1.3658,0.
```
Comment (samanpwbb) [inline browser.esm.js:3]: this looks wrong 🤔

File: scripts/gather-files.js [hunk 2]
```diff
@@ -8,11 +8,10 @@ const path = require('path');
  * @param {string} dir - Path to directory of files.
  * @return {array} – array of file contents
  */
-
+const hiddenFileRegex = /(^|\/)\.[^/.]/g;
 function gatherFiles(dir) {
-  return fs
-    .readdirSync(dir)
-    .map(f => fs.readFileSync(path.join(dir, f), 'utf8'));
+  const list = fs.readdirSync(dir).filter(item => !hiddenFileRegex.test(item));
```
Comment (tristen) [inline scripts/gather-files.js:13]: Oh nice catch. Couldn't this be much simpler and exclude anything that isn't .svg  i.e `.filter(item => item.match(/.svg$/))`?

**Review by tristen — CHANGES_REQUESTED** [review]
The silhouette and legibility look great. I just have some minor comments on some of the anchor points 

| Round `x` `y` coordinates | Superflous points | 
| --- | --- |
| Screen Shot 2021-09-27 at 3 39 40 PM | Screen Shot 2021-09-27 at 3 39 49 PM |

**Comment by samanpwbb** [comment]
Lol no, wait, I don't have permission to commit any more. One sec.

**Comment by tristen** [comment]
👉 https://github.com/mapbox/maki/pull/559

**Review by tristen — APPROVED** [review]
😭

**Pull request merged**